### DISCUSSION TITLE
Playtime: Ability to add automation envelopes to clips

### DISCUSSION THREAD

**Discussion by helgoboss** [helgoboss/helgobox#1041] [Ideas]

**Comment by helgoboss** [comment]
Just a remark: This feature would be quite large. Definitely not something that will make it in during the next 3 months. Also quite likely that it will be provided on a step-by-step basis. Below some things that would be involved (not complete).

## Automation playback

- Ability to actually play back automation curves
- Optional: Collaboration with Cockos to make setting FX parameters from audio thread possible

## Automation recording

- Ability to record automation into clips

## Automation editing

- Ability to add automatable parameters to a list and assign them an automation mode
- Ability to add edit existing automation curves in a sort of curve editor

**Comment by sterngit** [comment]
I really hope it will be developed in the future. I would like to play automations from clip in Playtime in order to control an other software. I just want the automation side no audio or midi in clip. Crossing fingers...

**Reply by helgoboss** [reply]
Hmm, but if you want to control another software, don't you need to use a message protocol such as MIDI? Which software?

**Reply by sterngit** [reply]
There is a new toy in town : https://www.grapes3d.com/
It is a clip based automation system in order to control spatial trajectories in other softwares. For example with midi automation I can write object movements in Azimuth Elevation Distance for Spat Revolution. https://www.flux.audio/project/spat-revolution/
It could be very usefull to play those movements from Playtime (and no need for Grapes).

**Comment by declanhalpin** [comment]
Hi Benjamin - this would be in my top 2 or 3 wishes for future releases.  Would it be any more feasible if it is limited to items already mapped by ReaLearn?
